src/main/resources/db/changelog/changes/01-create-table-books.yaml

```diff
@@ -36,7 +36,7 @@ databaseChangeLog:
                     nullable: false
               - column:
                   name: is_deleted
-                  type: boolean
+                  type: bit
```

Comment:
Why did you decide to change this type?

Reply:
I had a problem with boolean type when these tables were deleted accidentally. Type 'boolean' always was parsed as type 'bit' or 'tinyint'. Even if I used SQL in MySQL for creating table.

Reply:
Please, check if you didn't, which type preferable 'bit' or 'tinyint' for boolean.

Reply:
bit: storage size is 1 bit, possible values are 0 or 1
tinyint: storage size is 1 byte, possible values are 0-255